boards/DCM/Test/Src/Test_StateMachine.cpp

```diff
@@ -399,4 +409,147 @@ TEST_F(
     }
 }
 
+// DCM-8
+TEST_F(
+    DcmStateMachineTest,
+    regen_allowed_only_when_going_faster_than_5kph_and_airs_closed)
+{
+    SetInitialState(App_GetDriveState());
+
+    // Turn the DIM start switch on to prevent state transitions in
+    // the drive state.
+    EXPECT_TRUE(App_BuzzerSignals_IsOn(world));
+    App_CanRx_DIM_SWITCHES_SetSignal_START_SWITCH(
+        can_rx_interface, CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE);
+    ASSERT_EQ(
+        App_GetDriveState(),
+        App_SharedStateMachine_GetCurrentState(state_machine));
```

Comment:
don't think this adds anything since we are expected to be in the drive state. The state machine hasn't been ticked yet